src/luascript.cpp

```diff
@@ -1510,6 +1510,10 @@ void LuaScriptInterface::registerFunctions()
 	registerEnum(ITEM_ATTRIBUTE_CHARGES)
 	registerEnum(ITEM_ATTRIBUTE_FLUIDTYPE)
 	registerEnum(ITEM_ATTRIBUTE_DOORID)
+	registerEnum(ITEM_ATTRIBUTE_DECAYTO)
```

Comment:
doesn't these belong to the attr PR? 😄

Reply:
Nope, not really. Those are just missing enums, they are not used in `/attr`.